stable/ark-server/templates/configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ark-server-config
+  namespace: {{ .Values.namespace.name }}
+data:
+{{ (tpl (.Files.Glob "configuration/*").AsConfig .) | indent 2 }}
```

Comment:
I think there is a problem with this approach because I have to specify the namespace:
```
configuration:
  apiVersion: ark.heptio.com/v1
  kind: Config
  metadata:
    namespace: heptio-ark
```
And I can't do that in `values.yaml`
What do you think?